browser/android/background_video/background_video_playback_tab_helper.cc

```diff
@@ -55,6 +104,17 @@ bool IsBackgroundVideoPlaybackEnabled(content::WebContents* contents) {
 
   return true;
 }
+
+bool IsPictureInPictureForYouTubeVideosEnabled(content::WebContents* contents) {
```

Comment:
is this distinct from `IsBackgroundVideoPlaybackEnabled`? Would we inject this even if `IsBackgroundVideoPlaybackEnabled` returns false?